components/Common/Codebox/index.tsx

```diff
@@ -15,7 +15,9 @@ const Codebox: FC<CodeboxProps> = async props => {
   if (!isValidElement(props.children)) return null;
 
   const code = props.children.props.children.trim();
-  const lang = props.children.props.className.replace('language-', '').replace('mjs', 'js');
+  const lang = props.children.props.className
```

Comment:
nit: it seems wrong to use classname (but if that's how some tool works, sure)

Reply:
i have test it manually 😇
"I am the test runner"